### PR TITLE
Add kernel patch to fix GUI installer crash due to mmap issue

### PR DESCRIPTION
**Pull request by anphel31**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [ ] Ready to merge

---

###### Summary 

This resolves a crash in the graphical GUI ISO installer application.

###### Change Log  

- Add kernel patch
- Update related kernel package release numbers

###### Does this affect the toolchain?  

YES
**NO**

###### Associated issues  

- #xxxx

###### Links to CVEs  
- https://nvd.nist.gov/...

###### Test Methodology

- Local package and ISO build. Booted ISO in hyper-v, ran graphical installer, and didn't see the crash.

### CODE REVIEW

File: SPECS/kernel-headers/kernel-headers.spec [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Summary:        Linux API header files
 Name:           kernel-headers
 Version:        5.4.83
-Release:        2%{?dist}
+Release:        3%{?dist}
```
Comment (PawelWMS) [inline SPECS/kernel-headers/kernel-headers.spec:4]: I think we also need to update `kernel-hyperv` the same way.

**Review by PawelWMS — APPROVED** [review]

File: SPECS/kernel-headers/kernel-headers.spec (hunk 1 above)
Comment (anphel31) [reply SPECS/kernel-headers/kernel-headers.spec:4]: Thanks, fixed

File: SPECS/hyperv-daemons/hyperv-daemons.spec [hunk 2]
```diff
@@ -9,7 +9,7 @@
 Summary:        Hyper-V daemons suite
 Name:           hyperv-daemons
 Version:        5.4.83
-Release:        2%{?dist}
+Release:        3%{?dist}
```
Comment (christopherco) [inline SPECS/hyperv-daemons/hyperv-daemons.spec:12]: In general, it is not required to update the release number for `hyperv-daemons` or `kernel-hyperv` when updating `kernel.spec`.

File: SPECS/kernel-hyperv/kernel-hyperv.spec [hunk 3]
```diff
@@ -3,7 +3,7 @@
 Summary:        Linux Kernel optimized for Hyper-V
 Name:           kernel-hyperv
 Version:        5.4.83
-Release:        2%{?dist}
+Release:        3%{?dist}
```
Comment (christopherco) [inline SPECS/kernel-hyperv/kernel-hyperv.spec:6]: In general, it is not required to update the release number for `hyperv-daemons` or `kernel-hyperv` when updating `kernel.spec`.

File: SPECS/hyperv-daemons/hyperv-daemons.spec (hunk 2 above)
Comment (anphel31) [reply SPECS/hyperv-daemons/hyperv-daemons.spec:12]: Sounds good. Would it be better to revert this change or keep as-is? Do you have a preference?

Comment (christopherco) [reply SPECS/hyperv-daemons/hyperv-daemons.spec:12]: Please revert so release=2. Less overall delta.

Comment (anphel31) [reply SPECS/hyperv-daemons/hyperv-daemons.spec]: Reverted

File: SPECS/kernel-hyperv/kernel-hyperv.spec (hunk 3 above)
Comment (anphel31) [reply SPECS/kernel-hyperv/kernel-hyperv.spec]: Reverted

**Review by christopherco — APPROVED** [review]

**Pull request merged**